book/src/gobject_values.md

```diff
@@ -0,0 +1,33 @@
+# Generic Values
+
+Some GObject-related functions rely on generic values for their arguments or return parameters.
+Enumerators in C are not as powerful as the one in Rust, which is why [`glib::Value`](http://gtk-rs.org/docs/glib/value/struct.Value.html) is used for this.
```

Comment:
Enums / enumerations would be the correct word. An enumerator would be a different word for an iterator

Reply:
TIL, thanks!